examples/BasicUsage/README.md

```diff
@@ -0,0 +1,52 @@
+# Walkthrough
+This example walks through the necessary steps to create an ASP.NET Core 2.0 MVC application to Kubernets cluster with `Application Insights for Kubernetes` on.
+
+_Note: This is an example, not a best practice. You will need to consider more around the security aspects like how to protect the application insights instrumentation key and so on._
```

Comment:
How about the following:

This is a simple example that does not follow all best practices, including security-related best practices. E.g. Application Insights instrumentation key is not adequately protected (it should be deployed as a secret)